package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "jwks-rsa",
-  "version": "1.9.0",
+  "version": "2.0.0-beta",
```

Comment:
Not sure how this would affect the `auth0` release/package management, if we're changing version in their `master` branch

Reply:
It is a proposal for now, I actually don't know the release process here and need a guidance.

Reply:
we should remove the version proposal from the changeset.